### DISCUSSION TITLE
mcmanager.exe refuses to launch for non-admin on Windows

### DISCUSSION THREAD

**Discussion by JawnnyB** [nextgenhealthcare/connect#5689] [Q&A]
Anyone know why the mcmanager executable consistently shows 'Required elevation' errors on Windows for non local administrators?  I've tried many variations of Full Control to the mcmanager.exe, containing folder, and service but it refuses to run.

MCAL runs successfully for these users as well as Mirth Connect itself -- seems like just the Server Manager doesn't allow running as a non-admin?

**Comment by tonygermano** [comment]
MCAL is an end- user tool, and should not be run with an admin account. If by "Mirth Connect itself" you mean the server component, that also should not be run as an admin account as a best practice.

I don't use mcmanager, but I believe the primary function of that utility is to control stopping and starting the Windows service, and I believe that requires admin privileges,

**Reply by JawnnyB** [reply]
@tonygermano thanks for the response!  I was just adding a screenshot from Task Manager showing that 'UAC Virtualization' is not allowed for the mcmanager executable.  I think you're correct that it's primary use is start/stop of the Windows service so that would make sense.  

<img width="1441" alt="Screenshot 2023-02-20 at 16 04 09" src="https://user-images.githubusercontent.com/21296202/220198507-11b7a36b-5796-478a-967b-ae7b9b046104.png">
